.travis.yml

```diff
@@ -0,0 +1,47 @@
+# Config file for automatic testing at travis-ci.org
+
+language: python
+
+python:
+  - 2.7
+  - 3.5
+
+env:
+  - TOXENV=django18
+  - TOXENV=django19
```

Comment:
Thanks @e-kolpakov and @bjacobel. I ran into issues with Django 10 when I added the Django view tests. Now that I've converted to pytest it all runs smoothly so I've re-enabled this.